man/wifibox-alpine.5

```diff
@@ -619,91 +619,101 @@ additional, often proprietary firmware files have to be placed under
 .Pa /lib/firmware
 for activation.
 .Pp
+A list of wireless cards supported by the drivers is as follows.  The
```

Comment:
Thanks! Merging this PR would be an improvement. Yet I believe that it could be further improved, if targeting people with my level of reading skills. I made an attempts at including a couple of clues like this:

```diff
From 7f7ad0a3b686b567b6d1174f3b59bdf9d27462cd Mon Sep 17 00:00:00 2001
From: cos <cos>
Date: Mon, 27 May 2024 17:03:24 +0200
Subject: [PATCH] fixup! man: add hints about the availability of hardware
 support

---
 man/wifibox-alpine.5 | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/man/wifibox-alpine.5 b/man/wifibox-alpine.5
index 9d1ec64..1d1da7d 100644
--- a/man/wifibox-alpine.5
+++ b/man/wifibox-alpine.5
@@ -620,10 +620,12 @@ additional, often proprietary firmware files have to be placed under
 for activation.
 .Pp
 A list of wireless cards supported by the drivers is as follows.  The
-kernel modules that depend on specific firmware files are marked by
-name at the end of each entry, otherwise they should be working.  Note
-that this list might not be accurate and included here for information
-only.
+kernel modules that depend on specific firmware files, which might be
+added by building this port from source, are marked by name at the end
+of each entry, otherwise they should be working.  For some drivers
+there are makefile options to help in creating builds including the
+non-distributable files.  Note that this list might not be accurate
+and included here for information only.
 .Pp
 .Bl -tag -width Ds -offset indent -compact
 .It ADMTek/Infineon AMD8211A
-- 
2.43.0
```

That altered branch has also been pushed to [git.netizen.se/freebsd-wifibox-alpine](https://git.netizen.se/freebsd-wifibox-alpine/commit/?h=fix/man/highlight-driver-dependencies) (which should be possible to add as a git remote).

What do you think? Would such extra clarity make the manual overly verbose?

Reply:
I agree that some more clarity can be useful around when each of the firmware files are included.  I kept the text somewhat abstract intentionally to make it less fragile to updates and changes in the implementation details.  I have now amended the change based on your suggestion and hopefully this version unites our thoughts about the desired outcomes.

Reply:
That new next is way better than my attempt. I like it! Higher information density, while also succeeding at giving clear hints on how to think and where to look. Without making the words insult the intelligence of the initiated reader (like my suggestion likely would).

Great phrasing and great work! +1. Approved. Ship it!

Reply:
Thank you for your feedback.